sdk/eventhub/azure-eventhubs/azure/eventhub/_eventprocessor/__init__.py

```diff
@@ -1,4 +1,4 @@
 # --------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------------
```

Comment:
Strange? :)

Reply:
Ha, don't know why we made this change ;-)